ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -887,15 +887,26 @@ let vdi_copy_fun __context dbg vdi_map remote is_intra_pool remote_vdis so_far
     else
       (not is_intra_pool) || dest_sr <> vconf.sr
   in
+  let vbds = Db.VDI.get_VBDs ~__context ~self:vconf.vdi in
+  let _vm =
```

Comment:
This code is inside `vdi_copy_fun`, which is only called from `migrate_send'`, which knows the `vm`. It would be easier to just add a `vm` param to `vdi_copy_fun`.